pkgs/development/python-modules/mung/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, lxml
+, matplotlib
+, midiutil
+, numpy
+, pytestCheckHook
+, scikitimage
+, scikit-learn
+}:
+
+buildPythonPackage rec {
+  pname = "mung";
+  version = "unstable-2022-07-10";
```

Comment:
explicit format

Reply:
What is that? Is this documented somewhere?

Reply:
> format ? "setuptools": Format of the source. Valid options are "setuptools", "pyproject", "flit", "wheel", and "other". "setuptools" is for when the source has a setup.py and setuptools is used to build a wheel, flit, in case flit should be used to build a wheel, and wheel in case a wheel is provided. Use other when a custom buildPhase and/or installPhase is needed.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

Reply:
Ah, I thought the comment was about the version string somehow …